tracing-subscriber/src/filter/subscriber_filters/mod.rs

```diff
@@ -156,8 +162,18 @@ struct DebugCounters {
     in_interest_pass: Cell<usize>,
 }
 
+#[cfg(debug_assertions)]
+impl DebugCounters {
+    const fn default() -> Self {
```

Comment:
similarly, i have a slight preference for calling this `new` rather than `default`.